.gitignore

```diff
@@ -1,3 +1,4 @@
 build/
 env/
 .pytest_cache
+usr/*
```

Comment:
Question: can we use a more descriptive path than `usr` or is it fixed?

Reply:
it is the equivalent to the global usr dir ;)

we create it atm within the building workspace to have a clean setup

Reply:
I feel like `usr` is the default directory for binaries in the unix world. Sure we could change it to be more descriptive `vendor` for example?

Reply:
As @sebbrandt87 noted, this is added to gitignore cause the build process may create such folder and you don't want to commit such mutations later on, by accident. Renaming that to anything else does not make sense to me. 

How about we add a commenting line here explaining that? Apparently two people already got confused already, so worth the invest :D